src/NSwag.Generation.WebApi/Processors/OperationParameterProcessor.cs

```diff
@@ -495,14 +495,6 @@ private OpenApiParameter AddBodyParameter(OperationProcessorContext context, str
                 {
                     operationParameter.Default = defaultValue;
                 }
-                else if (operationParameter.Schema.HasReference)
```

Comment:
I think removing it is wrong, but you can just synchronize it with the non-legacy (ASP.NET Core) implementation: 

![image](https://user-images.githubusercontent.com/2603405/99941890-0f861f80-2d6f-11eb-8853-7517147b0e30.png)

https://github.com/RicoSuter/NSwag/blob/master/src/NSwag.Generation.AspNetCore/Processors/OperationParameterProcessor.cs#L450

And then set AllowReferencesWithProperties to false

Reply:
Thanks for the input, but how should this be synchronized? Is there some way to recognize if the other place already added schema OneOf wrapper?